Missions/__AR/11_cont/AI.src

```diff
@@ -2534,13 +2534,49 @@ Every 0$2+0$0.6 do
                     num_sol = Rand(1,2);
                end;
 
-          temp_list = CreateUnitsWithClass(num_sci,class_scientistic,side);
-          human_list = human_list ^ temp_list;
-          ai_attacking_scientists = Replace(ai_attacking_scientists,base,temp_list);
+          // if reservesCount > 0 then
+          //      .... svi CreateUnitsWithClass
+          //      reservesCount = reservesCount - temp_list (the list will decay into it's size)
+          // end
+
+          if russian_reserve_troops > 0 then
+             begin
+                  if logsenabled then
+                     display_strings = ['Spawning russian scrientists! '];
```

Comment:
dont use `display_strings` but rather write this out to a log file. using display_strings is useless because all I get to see is the latest `display_strings` call :/